setup_gui.py

```diff
@@ -342,102 +389,116 @@ def add_cm_tab(self, cm_name: str):
             cav_button_hlayout.addStretch()
-            
-            cav_vlayout.addLayout(cav_desamp_hlayout)
+
+            cav_vlayout.addLayout(cav_amp_hlayout)
             cav_vlayout.addLayout(cav_button_hlayout)
             cav_vlayout.addWidget(cav_widgets.status_label)
-            all_cav_layout.addWidget(cav_groupbox,
-                                     0 if cav_num in range(1, 5) else 1,
-                                     (cav_num - 1) % 4)
+            all_cav_layout.addWidget(
+                cav_groupbox, 0 if cav_num in range(1, 5) else 1, (cav_num - 1) % 4
+            )
 
 
 class SetupGUI(Display):
     def ui_filename(self):
-        return 'setup_gui.ui'
-    
+        return "setup_gui.ui"
+
     def __init__(self, parent=None, args=None):
         super(SetupGUI, self).__init__(parent=parent, args=args)
         self.threadpool = QThreadPool()
         print(f"Max thread count: {self.threadpool.maxThreadCount()}")
-        
+
         self.checkThreadTimer = QTimer(self)
         # I think this is 1 second?
         self.checkThreadTimer.setInterval(1000)
         self.checkThreadTimer.timeout.connect(self.update_threadcount)
         self.checkThreadTimer.start()
-        
-        self.settings = Settings(ssa_cal_checkbox=self.ui.ssa_cal_checkbox,
-                                 auto_tune_checkbox=self.ui.autotune_checkbox,
-                                 cav_char_checkbox=self.ui.cav_char_checkbox,
-                                 rf_ramp_checkbox=self.ui.rf_ramp_checkbox)
-        
+
+        self.settings = Settings(
+            ssa_cal_checkbox=self.ui.ssa_cal_checkbox,
+            auto_tune_checkbox=self.ui.autotune_checkbox,
+            cav_char_checkbox=self.ui.cav_char_checkbox,
+            rf_ramp_checkbox=self.ui.rf_ramp_checkbox,
+        )
+
         self.linac_widgets: List[Linac] = []
         for linac_idx in range(0, 4):
-            self.linac_widgets.append(Linac(f"L{linac_idx}B", linac_idx,
-                                            sc_linac_utils.LINAC_TUPLES[linac_idx][1],
-                                            settings=self.settings,
-                                            parent=self))
-        
-        self.linac_widgets.insert(2, Linac("L1BHL", 1, sc_linac_utils.L1BHL,
-                                           settings=self.settings, parent=self))
-        
-        self.linac_aact_pvs: List[PV] = [PV(f"ACCL:L{i}B:1:AACTMEANSUM") for i in range(4)]
-        
+            self.linac_widgets.append(
+                Linac(
+                    f"L{linac_idx}B",
+                    linac_idx,
+                    sc_linac_utils.LINAC_TUPLES[linac_idx][1],
+                    settings=self.settings,
+                    parent=self,
+                )
+            )
+
+        self.linac_widgets.insert(
+            2,
+            Linac(
+                "L1BHL", 1, sc_linac_utils.L1BHL, settings=self.settings, parent=self
+            ),
```

Comment:
Why a ',' here?

Reply:
No idea, Black put it there lol I think I can remove it